src/Standards/Squiz/Docs/Classes/SelfMemberReferenceStandard.xml

```diff
@@ -33,10 +59,6 @@ self<em> </em>::<em> </em>foo();
         <![CDATA[
 class Foo
 {
-    public static function bar()
-    {
-    }
-
     public static function baz()
```

Comment:
Why did you choose to remove the `bar()` methods ? For this specific error, it kind of makes sense to have the function declaration being referenced available in the code sample (though possibly this could be condensed to a one-liner with the braces on the declaration line).

I also wonder if the `static` in the function signatures should be removed, as it could give the impression - same as the updated description -, that this is about `static` methods only, while that's not the case.

Reply:
Yes, please update the function signature.

Re: the removed method - I don't see any harm in these being there and it makes it clearer what this is about, so I see no reason why it should be removed.

Note: comparing this to the `UselessOverridingMethod` sniff is an invalid comparison. This sniff is about references to methods in the same class, while the `UselessOverridingMethod` sniff is about methods being overloaded _from another class_, so not the same thing.

Reply:
Done